build/build-bundle.js

```diff
@@ -83,6 +83,11 @@ async function buildBundle(entryPath, distPath, opts = {minify: true}) {
     ...Runner.getAuditList().map(gatherer => `../audits/${gatherer}`),
   ];
 
+  // Added for smokerider test (used by byte-efficiency).
+  if (isLightrider(entryPath)) {
+    dynamicModulePaths.push('../audits/script-elements-test-audit.js');
```

Comment:
isn't it already in there from `getAuditList` above?

Reply:
lol yup! Wonder why I get "can't resolve" errors in LR then...

```
{
  "runtimeError": {
    "code": "UNKNOWN_ERROR",
    "message": "Unknown error encountered with message 'Could not dynamically require \"script-elements-test-audit\". Please configure the dynamicRequireTargets or/and ignoreDynamicRequires option of @rollup/plugin-commonjs appropriately for this require call to work.'"
  }
}
```